apps/bfd-model/bfd-model-dsl-codegen/bfd-model-dsl-codegen-plugin/src/test/java/gov/cms/model/dsl/codegen/plugin/model/ModelUtilTest.java

```diff
@@ -19,6 +21,10 @@ public class ModelUtilTest {
   /** Test methods that parse class and package names from strings. */
   @Test
   public void testClassNameParsing() {
+    assertFalse(ModelUtil.isValidFullClassName(null));
+    assertFalse(ModelUtil.isValidFullClassName(""));
+    assertFalse(ModelUtil.isValidFullClassName("String"));
+    assertTrue(ModelUtil.isValidFullClassName("java.lang.String"));
     assertEquals("java.lang", ModelUtil.packageName("java.lang.String"));
     assertEquals("String", ModelUtil.className("java.lang.String"));
     assertEquals(ClassName.get(String.class), ModelUtil.classType("java.lang.String"));
```

Comment:
These seem like they should be separate tests, being that it's testing several different methods

Reply:
The different methods are simple variations doing the same thing (parsing class names) so breaking them into separate methods would be excessive.